__tests__/app.test.js

```diff
@@ -142,6 +143,71 @@ describe('GET /api/articles/:article_id', () => {
       });
   });
 });
+
+describe('GET /api/articles/:article_id/comments', () => {
+  test('returns array of 11 comments for  article_id 1', () => {
+    return request(app)
+      .get('/api/articles/1/comments')
+      .expect(200)
+      .then((response) => {
+        const { comments } = response.body;
+        expect(comments.length).toBe(11);
+      });
+  });
+  test('returns array of 0 comments for  article_id 2', () => {
+    return request(app)
+      .get('/api/articles/2/comments')
+      .then((response) => {
+        const { comments } = response.body;
+        expect(comments.length).toBe(0);
+      });
+  });
+  test('each comment object has requirest keys', () => {
```

Comment:
Just a typo here in the test description, probably 'required' instead of 'requirest'.